src/spiceypy/tests/test_wrapper.py

```diff
@@ -6304,11 +6304,20 @@ def test_pckopn_pckw02_pckcls():
 
 def test_pckcov():
     ids = spice.cell_int(1000)
+    temp_obj = ids[0]
```

Comment:
this line cause the test to fail, I don't think it's necessary so removing it should fix things